ddtrace/tracer/writer.go

```diff
@@ -104,6 +104,16 @@ func (h *agentTraceWriter) flush() {
 	h.payload = newPayload()
 }
 
+var logWriter io.Writer = os.Stdout
```

Comment:
Can we add one line of doc?
```suggestion
// logWriter specifies the output target of the logTraceWriter; replaced in tests.
var logWriter io.Writer = os.Stdout
```